src/components/Rating/Rating.js

```diff
@@ -0,0 +1,71 @@
+class StarRating extends HTMLElement {
```

Comment:
I am not too sure about what this code does.
It would be nice if we could rewrite it to a React component, preferably writing it as a functional component using React hooks.